Controller/adminhtml/Checkout/Index.php

```diff
@@ -0,0 +1,253 @@
+<?php
+
+namespace Affirm\Telesales\Controller\Adminhtml\Checkout;
+
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\Controller\Result\JsonFactory;
+use Magento\Framework\Registry;
+use Magento\Framework\HTTP\ZendClientFactory;
+use Magento\Sales\Api\OrderRepositoryInterface;
+use Magento\Sales\Api\Data\OrderStatusHistoryInterface;
+use Magento\Framework\App\ProductMetadataInterface;
+use Magento\Framework\Module\ResourceInterface;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Astound\Affirm\Model\Config as ConfigAffirm;
+use Astound\Affirm\Model\Ui\ConfigProvider;
+use Affirm\Telesales\Model\Adminhtml\Checkout;
+
+class Index extends Action
+{
+    /**#@+
+     * Define constants
+     */
+    const API_STORE_CHECKOUT_PATH = '/api/v2/checkout/store';
+    const API_STORE_RESEND_PATH = '/api/v2/checkout/resend';
+    const CONFIRMATION_URL_ACTION = 'POST';
+    const CHECKOUT_FLOW_TYPE = 'In-Store';
```

Comment:
do we not have a flow type called telesales?

Reply:
removing unused constants